account_fiscal_year_reopen/README.rst

```diff
@@ -0,0 +1,60 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+==================
+Reopen Fiscal Year
+==================
+
+This module allows to reopen a fiscal year after it has been closed.
+
+Odoo advises to make a back-up of a database before closing a fiscal year, and
+to restore the database if the user regrets this decision. This is not a real
+solution as on restore all changes to the database made in the meantime will
+be lost.
+
+Of course a fiscal year can easily be reopened by resetting the state of the
+fiscal year to 'draft' using a postgress client like psql. This solution is
+dependant on system administrators. This module gives the possibility to
+financial managers. (Of course the sysadmin still needs to install this
+module ;-)).
+
+Configuration
+=============
+
+No special configuration is needed for this module.
+
+Known issues
+============
+
+None.
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/{project_repo}/issues>`_.
```

Comment:
You haven't replace here the markers by its values, and you can use the newest template for this: https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst